test/infer/test_svi.py

```diff
@@ -232,3 +232,59 @@ def guide():
     svi = SVI(model, guide, optim.Adam(1), Trace_ELBO())
     with pytest.warns(UserWarning, match="SVI does not support models with discrete"):
         svi.run(random.PRNGKey(0), 10)
+
+
+def test_normal_normal():
```

Comment:
is this a sufficient integration test? are there more tests that can be borrowed from pyro apart from the complex pyramid test? maybe the chain test? wdyt?

Reply:
There are some other tests for [trace graph elbo](https://github.com/pyro-ppl/pyro/blob/dev/tests/integration_tests/test_tracegraph_elbo.py) but they require baseline. I think without baseline things will still work, so I'll a couple more.

Reply:
yeah these low dimensional examples should work fine without baselines (although they will require more steps)

Reply:
https://github.com/pyro-ppl/pyro/blob/d7687ae0f738bd81a792dabbb18a53c0fce73765/tests/integration_tests/test_conjugate_gaussian_models.py

Reply:
maybe this is a good test to port: https://github.com/pyro-ppl/pyro/blob/0decbe54f8660cbf453ef4915495d77f0d7fb454/tests/integration_tests/test_tracegraph_elbo.py#L331